l10n-pocket/en/footer.ftl

```diff
@@ -2,10 +2,11 @@
 # License, v. 2.0. If a copy of the MPL was not distributed with this
 # file, You can obtain one at https://mozilla.org/MPL/2.0/.
 
-### URL all pages of https://www-dev-pocket.allizom.org/
+### URL all pages of https://dev.tekcopteg.com/
 
 
 # pocket/includes/footer.html
+# pocket/includes/platform-footer.html
```

Comment:
These comments aren't bound to any specific string / message. I'm not sure what the exact purpose is for these, but if they are to describe the file that's associated with a group of strings, then they should appear as a group level comment (`##`) preceding the strings that are related.